firebase-app-distribution/src/main/java/com/google/firebase/appdistribution/CheckForUpdateClient.java

```diff
@@ -150,24 +150,7 @@ private boolean isNewerBuildVersion(AppDistributionReleaseInternal latestRelease
   @VisibleForTesting
   boolean isInstalledRelease(AppDistributionReleaseInternal latestRelease) {
     if (latestRelease.getBinaryType().equals(BinaryType.APK)) {
-      Context context = firebaseApp.getApplicationContext();
-      PackageInfo metadataPackageInfo;
-      try {
-        metadataPackageInfo =
-            context
-                .getPackageManager()
-                .getPackageInfo(context.getPackageName(), PackageManager.GET_META_DATA);
-      } catch (PackageManager.NameNotFoundException e) {
-        return false;
-      }
-
-      String externalCodeHash = extractApkCodeHash(metadataPackageInfo);
-
-      if (externalCodeHash == null) {
-        return false;
-      }
-
-      return externalCodeHash.equals(latestRelease.getCodeHash());
+      return latestAndInstalledReleaseSameCodeHash(latestRelease);
```

Comment:
Since this method returns a boolean, we should probably follow the convention of asking a question as the name.

For example:
hasSameCodeHashAsInstallledRelease()